google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugSettingsEditor.java

```diff
@@ -47,7 +60,21 @@ protected JComponent createEditor() {
   }
 
   @Override
-  protected void resetEditorFrom(CloudDebugRunConfiguration runConfiguration) {
-    settingsPanel.setProjectName(runConfiguration.getCloudProjectName());
+  protected void resetEditorFrom(@NotNull CloudDebugRunConfiguration runConfiguration) {
+    String projectId = runConfiguration.getCloudProjectName();
```

Comment:
I'd add a comment / todo on this line just to make it clear why you are setting projectId to projectName

Reply:
agree.